File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    let num = ["zero", "one", "two", "three", "four", "five", "six", "seven","eight","nine"] 
    var lst = s
    
    for (let i = 0; i <num.length; i++) {
        let arr = lst.split(num[i]);
        lst = arr.join(i)
    }
    return Number(lst);
}

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    # 각 숫자에 대한 단어
    num = {"zero":"0","one":"1","two":"2","three":"3","four":"4","five":"5","six":"6",
           "seven":"7","eight":"8", "nine":"9"}
    for key in num:
        s = s.replace(key, num[key])
    return int(s)